syftlib/src/main/java/org/openmined/syft/execution/Plan.kt

```diff
@@ -12,10 +12,24 @@ import java.io.FileOutputStream
 
 private const val TAG = "syft.processes.Plan"
 
+/** 
+ * The Plan Class contains functions to load a Pytorch model from a TorchScript and
+ * passing the training data through the forward function of the Pytorch Module.
+ * A Pytorch Module is simply a container that takes in tensors as input and returns 
+ * tensor after doing some computation.
+ */
 @ExperimentalUnsignedTypes
 class Plan(val planId: String) {
     private var pytorchModule: Module? = null
 
+    /**
+     * Loads a serialized TorchScript module from the specified path on the disk.
+     *
+     * @param model The model parameters.
+     * @param trainingBatch The training data.
```

Comment:
Training batch = Pair<IValue,IValue>

Reply:
Should I add that it is the training data along with the labels ?

Reply:
Yeah that can also be added. The syntactic description is also needed like it is a pair of ivalues of labels and training data

Reply:
Ok

Reply:
Made the changes